src/Data/List/Membership/DecPropositional.agda

```diff
@@ -7,12 +7,13 @@
 {-# OPTIONS --cubical-compatible --safe #-}
 
 open import Relation.Binary.Definitions using (Decidable)
-open import Relation.Binary.PropositionalEquality using (_≡_)
-open import Relation.Binary.PropositionalEquality.Properties using (decSetoid)
+open import Relation.Binary.PropositionalEquality.Core using (_≡_)
 
 module Data.List.Membership.DecPropositional
   {a} {A : Set a} (_≟_ : Decidable (_≡_ {A = A})) where
```

Comment:
We definitely have a definition `DecidableEquality A` somewhere in the library....

Reply:
It's in `Relation.Binary.Definitions`, which is conveniently already imported here

Reply:
NB. That definition clearly post-dates some parts of the library (Eg. `Axiom.UniquenessOfIdentityProofs`...) where the idiom `(_≟_ : Decidable (_≡_ {A = A}))` is still in force... so we should consider a separate issue/PR to tidy all of those up?

Reply:
I found only 5 or 6 such uses... so maybe it's not as bad as I thought!

Reply:
Er... make that more like 30! Oops...

Reply:
See #2354

Reply:
I'll wait until that PR gets merged, and then merge it in to this one.